im.dino.Dino.json

```diff
@@ -0,0 +1,76 @@
+{
+    "id": "im.dino.Dino",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "dino",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=gpg-agent",
+        "--share=network",
+        "--device=all",
+        "--talk-name=org.freedesktop.Notifications",
+        "--device=dri"
```

Comment:
This is a duplicate. There is already a `--device=all` specified.

Reply:
Actually, why does it even need `--device=all`?

Reply:
Camera